libs/design-system/src/atoms/inputs/selectInputs.tsx

```diff
@@ -40,7 +40,7 @@ export type SelectInputProps = {
    * The index of the initial value.
    * It's 0 by default.
    */
-  initialValue?: string;
+  selectedValue?: string;
```

Comment:
Why? It's the initialValue of the input before human interaction

Reply:
I think I need something broader than a mere initialValue because, as I switch the toggle, I also want the **status** of the profile saved in Notion to be injected in the *status entry* of the form.

![statusSelectedEntryNotChanging](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/bee498d5-ad8e-4f96-863d-9e7e630f9ce7)

That being said, my solution is deprecated. According to this SO answer, [I should use `value` as well as default value](https://stackoverflow.com/a/44787318/22003549).

Reply:
It's still initial according to me as you re-render the component. But I'm okay with what you did 👌